LeanPoly/Poly.lean

```diff
@@ -472,23 +474,35 @@ def product.fanout {p q r : Poly} (f : r ⟶ p) (g : r ⟶ q) : r ⟶ p × q :=
   }
 
 def product.leftUnitor.hom (p : Poly) : (𝟭 × p) ⟶ p where
-  onPos := λ (_Unit , ppos) ↦ ppos
-  onDir := λ (_Unit , _ppos) pdir ↦ .inr pdir
+  onPos := λ (() , ppos) ↦ ppos
+  onDir := λ (() , _ppos) pdir ↦ .inr pdir
 
 def product.leftUnitor.inv (p : Poly) : p ⟶ (𝟭 × p) where
   onPos := λ ppos ↦ (.unit , ppos)
   onDir := λ _ppos dir ↦
   match dir with
   | .inr pfib => pfib
-  
--- TODO:
--- def product.leftUnitor (p : Poly) : (𝟭 × p) ≅ p where
---   hom := product.leftUnitor.hom p
---   inv := product.leftUnitor.inv p
---   hom_inv_id := _
---   inv_hom_id := by {
---     _
---   }
+
+def product.leftUnitor.hom_inv_id : composemap (leftUnitor.hom p) (leftUnitor.inv p) = 𝟙 (𝟭 × p)
```

Comment:
It's a bit annoying to have to make composemap explicitly here, Lean can't figure it out from Poly's `Category` instance for some reason. It gives us the ability to unfold the definition and then simplify